apollo-gradle-plugin/src/main/java/com/apollographql/android/gradle/ApolloCodeGenInstallTask.java

```diff
@@ -19,7 +19,7 @@
 public class ApolloCodeGenInstallTask extends NpmTask {
   static final String NAME = "installApolloCodegen";
   private static final String INSTALL_DIR = "node_modules/apollo-codegen";
-  private static final String APOLLOCODEGEN_VERSION = "0.10.5";
+  private static final String APOLLOCODEGEN_VERSION = "0.10.8";
```

Comment:
Yeah, good call. Seems like a better fit as this is something the Compiler is "aware" of, albeit indirectly